collect_app/src/main/java/org/odk/collect/android/activities/FormEntryActivity.java

```diff
@@ -27,7 +27,9 @@
 import android.os.AsyncTask;
 import android.os.Bundle;
 import android.provider.MediaStore.Images;
-import android.support.v7.app.AlertDialog;
+import android.support.annotation.Nullable;
+import android.support.v4.view.ViewConfigurationCompat;
+import android.support.v7.app.AlertDialog;>>
```

Comment:
`>>`? Did this just sneak in?

Reply:
Does this build as is? Shouldn't that >> bork things?

Reply:
That  probably came in when i was resolving some conflicts on Github. Let me clean the code and commit it.